packages/meteor-accounts-saml/saml_server.js

```diff
@@ -172,7 +176,7 @@ WebApp.connectHandlers.use(connect.bodyParser()).use(function (req, res, next) {
 	}).run();
 });
 
-middleware = function (req, res, next) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['middleware' is defined but never used.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2313529360/issues/source?bid=3199634&fileBranchId=3199636#l179)